com.unity.formats.alembic/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 ### Fixed
+- Fixed a bug that caused an exception to be thrown when manually adding an AlembicStreamPlayer Component.
```

Comment:
Fixed a bug that caused an error when manually adding an AlembicStreamPlayer Component.